src/python-common/requirements.txt

```diff
@@ -1,6 +1,4 @@
 six
-pytest >=2.1.3,<5; python_version < '3.5'
-mock; python_version < '3.3'
```

Comment:
Hm. python-common might get used by ceph-volume for the drive groups integration. As c-v is still on Python 2, I'm keeping python-common on Python 2 as well.

Reply:
please note, c-v is packaged along with ceph-osd, and as we know ceph-osd is shipped as part of ceph. and ceph is now packaged with a hard dependency on python3 instead of python2. i don't see the reason why we need to have python2 as an option.

Reply:
dropping py2 here makes backporting unnecessary painful, we we have to do the py3->py2 migration when doing backports.

Reply:
we will have to do python3->python2 migration one day. otherwise we will have to keep python2 support forever. i am not sure which is more painful. if i have to choose, i'd go with option 1.

Reply:
also, please note, removing this does not change the way how we tests python-common. if it supports python2, it still supports it after this change.